.github/workflows/nf-test-arm.yml

```diff
@@ -71,15 +71,7 @@ jobs:
       matrix:
         shard: ${{ fromJson(needs.nf-test-changes-arm.outputs.shard) }}
         archProfile: [arm64]
-        profile: [conda, docker, singularity]
-        isMain:
-          - ${{ github.base_ref == 'master' || github.base_ref == 'main' }}
-        # Exclude conda and singularity on dev
-        exclude:
-          - isMain: false
-            profile: "conda"
-          - isMain: false
-            profile: "singularity"
```

Comment:
I usually keep that part, that way it's easier for me to keep track of updates coming from TEMPLATE to the `nf-test.yml` file, if the changes are minimal